recipe/meta.yaml

```diff
@@ -50,7 +50,7 @@ about:
     By adding an anaconda-project.yml to a project directory, a single 
     anaconda-project runcommand will be able to set up all dependencies 
     and then launch the project.
-  doc_url: http://anaconda-project.readthedocs.io
+  doc_url: https://anaconda-project.readthedocs.io/en/latest/
```

Comment:
I agree with the use of `https` here but I do not think you should follow the redirect. That's an implementation detail of readthedocs. The home page is the better choice IMO

Reply:
I got the opposite comment with my last PR in another feedstock. I will raise the issue in the slack channel, so we can come to an agreement.

Reply:
Redirect is removed.